src/components/CreateNewProjectWizard/ProjectName.js

```diff
@@ -37,11 +37,13 @@ class ProjectName extends PureComponent<Props, State> {
     showRandomizeTooltip: false,
   };
 
-  node: HTMLElement;
+  node: ?HTMLElement;
   timeoutId: number;
 
   componentDidMount() {
-    this.node.focus();
+    if (this.node) {
+      this.node.focus();
+    }
```

Comment:
hah, nice. Digging that switching to `ref` means that Flow now catches more issues